teamB/kimSungHoon/routes/userRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const userController = require('../controllers/userController');
+
+const router = express.Router();
+
+router.post('/signup', userController.signUp);
+
+router.get('/userPosting/:id', userController.userPosting);
+
+router.post('/like', userController.like);
```

Comment:
추후 RESTful API 세션을 통해 배울 내용입니다만 RESTful API에서 엔드포인트를 구성할 땐 동사가 아닌 명사로 작성하게 됩니다. 또한 해당 메서드가 엔드포인트로 드러나지 않게 수정해주세요!

routes/index.js 파일에서 postRouter로 이동할 때의 url이 /posts인 것을 고려하여 다음과 같이 작성해줍니다.

엔드포인트 요청 시 url은 다음과 같습니다.
http://localhost:3000/users/signup (post)
http://localhost:3000/users/signin (post)

또한 posts 데이터를 받아오는 엔드포인트는 postRouter.js 파일로 옮겨주세요!

좋아요 기능을 위해 필요한 추가 기능이 있습니다. 확인 후 추가해주세요!
 
```suggestion
router.post('/signup', userController.signUp);
router.post('/signin', userController.signUp); // 추가 예시
```